src/main/kotlin/nl/info/client/zgw/zrc/jsonb/GeoJSONGeometryToBeDeletedJsonbSerializer.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.client.zgw.zrc.jsonb
+
+import jakarta.json.bind.serializer.JsonbSerializer
+import jakarta.json.bind.serializer.SerializationContext
+import jakarta.json.stream.JsonGenerator
+import nl.info.client.zgw.zrc.model.GeoJSONGeometryToBeDeleted
+
+/**
+ * Custom JSONB serializer for [GeoJSONGeometryToBeDeleted] objects.
+ * Writes a `null` value to indicate that the geometry field should be deleted, as per the ZGW ZRC API specification.
+ */
+class GeoJSONGeometryToBeDeletedJsonbSerializer : JsonbSerializer<GeoJSONGeometryToBeDeleted> {
```

Comment:
Does it make sense to change `ToBeDeleted` to `DeleteOperation` or just `DeleteGeoJSONGeometryJsonbSerializer` ?

Reply:
did this now